website/src/app/core/state/data-distribution/data-distribution.state.ts

```diff
@@ -0,0 +1,142 @@
+import { HttpClient } from '@angular/common/http';
+import { Dataset, EMPTY_DATASET } from './../../models/dataset.model';
+import { Injectable } from '@angular/core';
+import { Computed, DataAction, StateRepository } from '@ngxs-labs/data/decorators';
+import { NgxsDataRepository } from '@ngxs-labs/data/repositories';
+import { State } from '@ngxs/store';
+import { Observable } from 'rxjs';
+import { pluck } from 'rxjs/operators';
+import { EMPTY_TABLE_DATA, EMPTY_TABLE_DATA_DIRECTORY, TableData, TableDataDirectory } from '../../models/table-data.model';
+
+interface DataDistributionsStateModel {
+  datasets: Dataset[];
+  currentDataset: Dataset;
+}
+
+const SUB_LABEL_FLAG = 'Tox lab flag';
+const ASSETS_DIRECTORY = 'assets/generated/aggregate-table-data.json';
+const SUB_LABEL = 'Drug';
+
+@StateRepository()
+@State<DataDistributionsStateModel>({
+  name: 'page',
```

Comment:
This might be the thing that is causing it to fail. You need a unique name here. (data-distributions)